source/documentation/index.md

```diff
@@ -68,7 +68,7 @@ Guidance on using the Developer Hub and GitHub. Including end points, access tok
 
 ***
 
-#Surely a title should be here?
+#Getting Started
```

Comment:
H2

Reply:
You also want a space between the # and the text as it can be parsed differently at times (may not be an issue here)